packages/batch-poster-monitor/index.ts

```diff
@@ -348,46 +397,54 @@ const monitorBatchPoster = async (childChainInformation: ChainInfo) => {
   // Get the last block of the chain
   const latestChildChainBlockNumber = await childChainClient.getBlockNumber()
 
-  if (!sequencerInboxLogs || sequencerInboxLogs.length === 0) {
-    // No SequencerInboxLog in the last 24 hours (time hardcoded in getDefaultBlockRange)
-    // We compare the "latest" and "safe" blocks, and check if any of the pending blocks contain any user-transactions
-    // NOTE: another way of verifying this might be to check the timestamp of the last block in the childChain chain to verify more or less if it should have been posted
-    const latestChildChainSafeBlock = await childChainClient.getBlock({
-      blockTag: 'safe',
-    })
+  const latestChildChainSafeBlock = await childChainClient.getBlock({
+    blockTag: 'safe',
+  })
+
+  const blocksPendingToBePosted =
+    latestChildChainBlockNumber - latestChildChainSafeBlock.number
+
+  const batchPostingTimeBounds = await getBatchPostingTimeBounds(
+    childChainInformation,
+    parentChainClient
+  )
 
-    const blocksPendingToBePosted =
-      latestChildChainSafeBlock.number < latestChildChainBlockNumber
+  const doPendingBlocksContainUserTransactions =
+    await checkForUserTransactionBlocks({
+      fromBlock: Number(latestChildChainSafeBlock.number + 1n), // start checking AFTER the latest 'safe' block
+      toBlock: Number(latestChildChainBlockNumber),
+      publicClient: childChainClient,
+    })
 
-    const doPendingBlocksContainUserTransactions =
-      await checkForUserTransactionBlocks({
-        fromBlock: Number(latestChildChainSafeBlock.number + 1n), // start checking AFTER the latest 'safe' block
-        toBlock: Number(latestChildChainBlockNumber),
-        publicClient: childChainClient,
-      })
+  const batchPostingBacklog =
+    blocksPendingToBePosted > 0n && doPendingBlocksContainUserTransactions
 
-    if (blocksPendingToBePosted && doPendingBlocksContainUserTransactions) {
+  if (!sequencerInboxLogs || sequencerInboxLogs.length === 0) {
+    // if alert situation
+    if (batchPostingBacklog) {
       alertsForChildChain.push(
         `No batch has been posted in the last ${
-          DEFAULT_TIMESPAN_SECONDS / 60 / 60
+          MAX_TIMEBOUNDS_SECONDS / 60 / 60
         } hours, and last block number (${latestChildChainBlockNumber}) is greater than the last safe block number (${
           latestChildChainSafeBlock.number
-        })`
+        }). Atleast 1 block must be posted every ${
```

Comment:
```suggestion
        }). At least 1 batch is expected to be posted every ${
```
Smol nitpick for accuracy in text

Reply:
Updated